### PR TITLE
Add Gunicorn configuration for Prometheus

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/lib/metrics.py [hunk 1]
```diff
@@ -27,86 +27,94 @@
...
+        gauge = prometheus_client.core.GaugeMetricFamily(
+            "c2cgeoportal_memory_cache_bytes",
+            "Memory used by cache",
+            labels=["pid", "hostname", "key"],
+            unit="bytes",
+        )
+
         elements = _get_memory_cache(all_=self.all)
         assert elements is not None
         result = []
```
Comment (vuilleumierc) [inline geoportal/c2cgeoportal_geoportal/lib/metrics.py:58]: it `result` still needed?

File: geoportal/c2cgeoportal_geoportal/lib/metrics.py [hunk 2]
```diff
@@ -27,86 +27,94 @@
...
-def _get_memory_cache(all_: bool) -> dict[str, list[tuple[dict[str, Any], float]]]:
-    values = (
-        [({"key": key}, get_size(value) / 1024) for key, value in list(MEMORY_CACHE_DICT.items())]
-        if all_
-        else []
-    )
-    values.append(({"key": "total"}, get_size(MEMORY_CACHE_DICT) / 1024))
+def _get_memory_cache(all_: bool) -> dict[str, list[tuple[str, float]]]:
+    values = [(key, get_size(value) / 1024) for key, value in list(MEMORY_CACHE_DICT.items())] if all_ else []
+    values.append(("total", get_size(MEMORY_CACHE_DICT)))
```
Comment (vuilleumierc) [inline geoportal/c2cgeoportal_geoportal/lib/metrics.py:70]: missing `/ 1024`?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/lib/metrics.py:70]: No, previously we were using kbytes, but Prometheus prefers bytes :-)

Comment (vuilleumierc) [reply geoportal/c2cgeoportal_geoportal/lib/metrics.py:70]: OK :+1: but then maybe the `/ 1024` on the previous line (69) should be removed? (sorry I edited my previous comment at the same time you were responding)

File: geoportal/c2cgeoportal_geoportal/lib/metrics.py (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/lib/metrics.py]: No, tanks :-)

**Review by vuilleumierc — APPROVED** [review]

**Pull request merged**